nengo/tests/test_examples.py

```diff
@@ -73,6 +81,8 @@ def assert_noexceptions(nb_file, tmpdir):
 @pytest.mark.filterwarnings("ignore:Creating new attribute 'memory_location'")
 def test_fast_noexceptions(nb_file, tmpdir):
     """Ensure that no cells raise an exception."""
+    pytest.importorskip("IPython", minversion="1.0")
```

Comment:
I'm not sure if this is relevant to this PR, but why is the `minversion="1.0"`. I think I'm confused about the relation between `Jupyter` and `IPython`. Please let me know if I shouldn't worry about this.

Reply:
Right yeah we should probably bump these up, I copied the line from elsewhere in this file.